bigframes/ml/llm.py

```diff
@@ -484,8 +487,11 @@ def _create_bqml_model(self):
             )
 
         if self.model_name not in _PALM2_EMBEDDING_GENERATOR_ENDPOINTS:
-            raise ValueError(
-                f"Model name {self.model_name} is not supported. We only support {', '.join(_PALM2_EMBEDDING_GENERATOR_ENDPOINTS)}."
+            warnings.warn(
+                f"Model name '{self.model_name}' is not supported. "
+                f"We currently support the following models: {', '.join(_PALM2_EMBEDDING_GENERATOR_ENDPOINTS)}. "
+                "However, model names can change, and the supported models may be outdated. "
```

Comment:
The last two lines are common, we can define it once as a constant

Reply:
The warning message is updated to be a template.